### ISSUE TITLE
No audio description for payment card video

### ISSUE THREAD

**Issue by nickcolley** [alphagov/govuk-design-system#905]
**WCAG Reference**: [1.2.5 Audio Description (Pre-recorded) (Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/audio-description-prerecorded.html)
**Issue ID**: DAC_Issue12
**URL**: https://design-system.service.gov.uk/patterns/payment-card-details/

## Screen Shot
There was no audio description available within the video to enable users that have difficulty seeing the screen to ascertain what was being relayed.

## Current Code Ref(s)
```html
<video role="region" aria-label="Card number validation flow example video, this video has no audio." controls="" muted=""> <source src="card-number.mp4" type="video/mp4"> </video>
```

## Screen reader comments
> “The video track on the page did not possess any audio. I was unable to understand what the video content was displaying as no audio description track was provided. Ensuring that an audio descriptive track describes what is occurring within the video will avoid any difficulty. This occurs using JAWS, NVDA and Voiceover for iPhone and Mac.”

## Solution
Ensure that an audio description is provided for users that cannot see what is occurring in the video.
For adding a ‘track’ to the video please refer to:
https://www.w3.org/WAI/WCAG21/Techniques/html/H96

**Comment by nickcolley** [comment]
It seems as though there is good explanation about this pattern above and below, worth checking if all context is covered in text.

If it's already covered in the text surrounding it could we indicate this in the label for the video?

I'm not sure adding a audio description track makes sense as it'd duplicate content.

**Comment by kellylee-gds** [comment]
Update the video's accessible name to make it clear that it's described by the content that surrounds it.
